clients/netcore/DataAPIClient.gen.cs

```diff
@@ -2548,6 +2782,70 @@ public partial class ReadValueDoubleModel
+        public System.DateTimeOffset Timestamp { get; set; }
+    
+        /// <summary>The timestamp at which the Value was created on the server.</summary>
+        [Newtonsoft.Json.JsonProperty("createdAt", Required = Newtonsoft.Json.Required.DisallowNull, NullValueHandling = Newtonsoft.Json.NullValueHandling.Ignore)]
+        public System.DateTimeOffset CreatedAt { get; set; }
+    
+        /// <summary>The actual data stored in this 'Value'</summary>
+        [Newtonsoft.Json.JsonProperty("values", Required = Newtonsoft.Json.Required.Default, NullValueHandling = Newtonsoft.Json.NullValueHandling.Ignore)]
+        public System.Collections.Generic.ICollection<float> Values { get; set; }
+    
+        /// <summary>The synchronisation sequence number. &lt;br /&gt;
+        /// <br/>The sequence defines a order for the Values within a ValueMetadata group
+        /// <br/>and is used for detection data-gaps on a receiving Node.</summary>
+        [Newtonsoft.Json.JsonProperty("syncNr", Required = Newtonsoft.Json.Required.DisallowNull, NullValueHandling = Newtonsoft.Json.NullValueHandling.Ignore)]
+        public long SyncNr { get; set; }
+    
+        /// <summary>The size of the 'Values' in byte.</summary>
+        [Newtonsoft.Json.JsonProperty("size", Required = Newtonsoft.Json.Required.DisallowNull, NullValueHandling = Newtonsoft.Json.NullValueHandling.Ignore)]
+        public long Size { get; set; }
+    
+    
+    }
+    
+    [System.CodeDom.Compiler.GeneratedCode("NJsonSchema", "10.5.2.0 (Newtonsoft.Json v12.0.0.0)")]
+    public partial class ReadValueIntModel 
+    {
+        /// <summary>ValueMetadataId, the ID of the data stream which the 'Value' belongs to</summary>
+        [Newtonsoft.Json.JsonProperty("valueMetadataId", Required = Newtonsoft.Json.Required.DisallowNull, NullValueHandling = Newtonsoft.Json.NullValueHandling.Ignore)]
+        public System.Guid ValueMetadataId { get; set; }
+    
+        /// <summary>The timestamp at which the Value was recorded on the trust agent/edge device.</summary>
+        [Newtonsoft.Json.JsonProperty("timestamp", Required = Newtonsoft.Json.Required.DisallowNull, NullValueHandling = Newtonsoft.Json.NullValueHandling.Ignore)]
+        public System.DateTimeOffset Timestamp { get; set; }
+    
+        /// <summary>The timestamp at which the Value was created on the server.</summary>
+        [Newtonsoft.Json.JsonProperty("createdAt", Required = Newtonsoft.Json.Required.DisallowNull, NullValueHandling = Newtonsoft.Json.NullValueHandling.Ignore)]
+        public System.DateTimeOffset CreatedAt { get; set; }
+    
+        /// <summary>The actual data stored in this 'Value'</summary>
+        [Newtonsoft.Json.JsonProperty("values", Required = Newtonsoft.Json.Required.Default, NullValueHandling = Newtonsoft.Json.NullValueHandling.Ignore)]
+        public System.Collections.Generic.ICollection<int> Values { get; set; }
```

Comment:
According to our user story it should be a `int` not `int[]`. We should first adapt the API and than re-generate client.